parser/parser_test.go

```diff
@@ -769,9 +775,9 @@ func (s *testParserSuite) TestBuiltin(c *C) {
 		{`select subdate("2011-11-11 10:10:10.123456", interval "11 10:10" day_minute)`, true},
 		{`select subdate("2011-11-11 10:10:10.123456", interval "11 10" day_hour)`, true},
 		{`select subdate("2011-11-11 10:10:10.123456", interval "11-11" year_month)`, true},
-		{`select adddate("2011-11-11 10:10:10.123456", 10)`, true},
-		{`select adddate("2011-11-11 10:10:10.123456", 0.10)`, true},
-		{`select adddate("2011-11-11 10:10:10.123456", "11,11")`, true},
+		{`select subdate("2011-11-11 10:10:10.123456", 10)`, true},
```

Comment:
I think it's a typo here. `adddate` has been tested [here](https://github.com/pingcap/tidb/pull/2368/files/e38dccc5293f3180052caee012983cdceaa96ed0#diff-27c45ca411f005e1b8796b12fb53e26cL725).